packages/shared/src/contexts/BuyCoresContext.tsx

```diff
@@ -98,165 +80,28 @@ export const BuyCoresContextProvider = ({
     id: string;
     value: number;
   }>();
-  const [paddle, setPaddle] = useState<Paddle>();
-  const isCheckoutOpenRef = React.useRef(false);
   const logRef = useRef<typeof logEvent>();
   logRef.current = logEvent;
 
-  const { data: prices } = useQuery(
-    transactionPricesQueryOptions({
-      user,
-      isLoggedIn,
-    }),
-  );
+  const { paddle, openCheckout } = usePaddlePayment({
```

Comment:
Not sure from all the changes, but just to make sure anything purchase Cores should not be available on iOS native. I don't think you touched that part of the code since it is managed in `useCanPurchaseCores`.

Reply:
Yep. Those areas should be untouched, only applied changes on how the Paddle is initialized and the pricing itself 🚀